code/index.html

```diff
@@ -1,22 +1,121 @@
-  </body>
-</html>
+
+<head>
+  <meta charset="utf-8" />
+  <title>mamzine magazine</title>
+  <link rel="stylesheet" href="./style.css">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Sora:wght@200;300&display=swap" rel="stylesheet">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+
+</head>
+
+<body>
+
+  <section class="full-container">
+    <header class="header-navigation-container">
+
+      <div>
+        <img class="logo" src="/code/img/Mz_logo.png" alt="logo">
+      </div>
+      <div class="header-navigation-container"></div>
+      <h1 class="site-name">mamzine</h1>
+      <ul class="nav-menu">
+        <li class="nav-item" style="display:inline"> <a href="#">Link</a> </li>
+        <li class="nav-item" style="display:inline"> <a href="#">Link</a> </li>
+        <li class="nav-item" style="display:inline"> <a href="#">Link</a> </li>
+        <li class="nav-item" style="display:inline"> <a href="#">Link</a> </li>
+
+      </ul>
+      </nav>
+      </div>
+  </section>
+  </header>
+
+  <!-- The Big News Section-->
+  <section class="big-news-container">
+
+    <div class="big-news-parent">
+      <img class="big-news-image" src="/code/img/queen of code.jpg" alt="code" />
+      <div class="text">
+        <h1>Queen of Code #1.</h1>
+        <p class="big-news-text">Computer programming once had much better gender balance than it does today. What went
+          wrong?</p>
+        <button class="read-more" type="button">We Need More Women In Tech! Read aboot the fab queens
+          on code</button>
+      </div>
+    </div>
+  </section>
+
+  <section class="news-cards-parent">
+    <div class="news-card-child">
+      <h3>Meet Betelhem Dessie, Ethiopia’s leading tech queen </h3>
+      <img class="news-img" src="/code/img/queen3.jpg" alt="women">
+      <p>I just might be a black Bill Gates in the making.I see it, I want it, I stunt, yellow-bone it
+        I dream it, I work hard, I grind 'til I own it.</p>
+    </div>
+
+    <div class="news-card-child">
+      <h3>Meet Betelhem Dessie, Ethiopia’s leading tech queen</h3>
+      <img class="news-img" src="img/queen3.jpg">
+      <p>I just might be a black Bill Gates in the making.I see it, I want it, I stunt, yellow-bone it
+        I dream it, I work hard, I grind 'til I own it. </p>
+    </div>
+
+    <div class="news-card-child">
+      <h3>Meet Betelhem Dessie, Ethiopia’s leading tech queen</h3>
+      <img class="news-img" src="/code/img/queen3.jpg">
+      <p>I just might be a black Bill Gates in the making.I see it, I want it, I stunt, yellow-bone it
+        I dream it, I work hard, I grind 'til I own it. </p>
+    </div>
+
+    <div class="news-card-child">
+      <h3>Meet Betelhem Dessie, Ethiopia’s leading tech queen</h3>
+      <img class="news-img" src="/code/img/queen3.jpg" alt="women">
```

Comment:
Referring to an image like this seems to works with Live Server but not with Netlify.